groups/groups.yaml

```diff
@@ -36,6 +36,11 @@ groups:
       - justinsb@google.com
       - spiffxp@google.com
       - thockin@google.com
+  - email-id: k8s-infra-rbac-cert-manager@kubernetes.io
```

Comment:
Really minor nit - we're eventually going to run out of characters, and this has a 15 char prefix.  

Maybe we should try to find a shorted prefix and elide some dashes like "certmanager" vs cert-manager" or event "certmgr" ?